p2p_wallet/SwiftUI/Renderable/ButtonListCellView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+import KeyAppUI
+
+struct ButtonListCellItem: Identifiable {
+    var id = UUID().uuidString
+    let leadingImage: UIImage?
+    let title: String
+    let action: () -> Void
+    let style: TextButton.Style
+    let trailingImage: UIImage?
+}
+
+extension ButtonListCellItem: Renderable {
+    func render() -> some View {
+        ButtonListCellView(
```

Comment:
is it essential to have here ButtonListCellView instead of raw NewTextButton

Reply:
Good point. I will substitute it with NewTextButton, no need to make additional views